models/Status.php

```diff
@@ -45,6 +45,11 @@ class Status extends Model
     const STATUS_COMPETE = 'complete';
     const STATUS_CANCELED = 'canceled';
 
+    const STATUS_NEW_COLOR = '#f1c40f';
```

Comment:
It would be better to name constants like this `STATUS_COLOR_***`.

Reply:
it seems right

Reply:
Great. Let's push a new commit with these changes.